services/lsst-dev.rst

```diff
@@ -256,6 +257,39 @@ Administrators may wish to note that the shared stack is automatically updated u
 
 .. _HTCondor pool: https://confluence.lsstcorp.org/display/DM/Orchestration
 
+.. _lsst-dev-testdata:
+
+Validation/Test Data Sets
+=========================
+
+As of early April 2019, there are two ``cron`` jobs that will update a set of validation data repositories and test data repositories.
```

Comment:
Do we care that it's “as of early April 2019”? Doesn't do any harm, but seems like unnecessary verbiage.